CHANGELOG.md

```diff
@@ -1,4 +1,27 @@
 ## 2.14.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-5.4.1166** 
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. Implemented support for dynamically updating ``customized_route_advertise`` in **aviatrix_aws_tgw_vpc_attachment**
+2. Implemented support for controller login SAML Endpoint in **aviatrix_saml_endpoint**
+3. Implemented **data_source_aviatrix_vpc**
```

Comment:
To maintain consistent formatting following previous release notes:

3. New data source to support referencing specific private/public subnets:
  - **data_source_aviatrix_vpc**